src/common/workspace-store.ts

```diff
@@ -7,10 +7,12 @@ import { broadcastMessage } from "../common/ipc";
 import logger from "../main/logger";
 
 export type WorkspaceId = string;
+export type ClusterId = string;
 
 export interface WorkspaceStoreModel {
+  workspaces: WorkspaceModel[];
   currentWorkspace?: WorkspaceId;
-  workspaces: WorkspaceModel[]
+  lastActiveClusterId?: ClusterId;
```

Comment:
Wait, does this mean that we only track the last active cluster once for the whole application (and per workspace)?

Reply:
Or are we only saving the last active cluster globally (still seems odd)?

Reply:
We're tracking last active cluster for each workspace. This field belongs to `Workspace` class.

Reply:
See my other (sniped) comment

Reply:
You mean this one? https://github.com/lensapp/lens/pull/1444#discussion_r526988672
Isn't it about same question about clarification of where `lastActiveClusterId` field is located?

Reply:
It is, I noticed that I had asked about the one located on `WorkspaceStoreModel` which seems strange to me. I would have though that it should be been on `WorskaceModel`

Reply:
Oh, that's true. Moved into `WorkspaceModel`.